src/styles/main.scss

```diff
@@ -20,88 +20,78 @@ $dark-color: black;
 $light-grey: lightgrey;
 $background-color: #f9f9f9;
 
-.view-wrapper {
+.react-digraph {
   height: 100%;
   width: 100%;
   margin: 0;
-  display: flex;
   box-shadow: none;
   background: $background-color;
-  transition: opacity 0.167s;
-  opacity: 1;
   outline: none;
   user-select: none;
 
-  > .graph {
-    align-content: stretch;
-    flex: 1;
+  .graph {
     width: 100%;
     height: 100%;
   }
 
-  .node {
-    .shape {
-      > use.node {
-        color: $primary-color;
-        stroke: $dark-color;
-        fill: $light-color;
-        filter: url(#dropshadow);
-        stroke-width: 0.5px;
-        cursor: pointer;
-        user-select: none;
-
-        &.hovered {
-          stroke: $primary-color;
-        }
-        &.selected {
-          color: $light-color;
-          stroke: $primary-color;
-          stroke-width: 1px;
-          fill: $primary-color;
-        }
-      }
-    }
+  .circle {
+    fill: $light-grey;
+  }
 
-    .node-text {
-      fill: $dark-color;
+  use {
+    &.node {
+      stroke: $dark-color;
+      fill: $light-color;
+      filter: url(#dropshadow);
+      stroke-width: 0.5px;
       cursor: pointer;
       user-select: none;
+
+      &:hover {
+        stroke: $primary-color;
+      }
+
       &.selected {
-        fill: $light-color;
-        stroke: $light-color;
+        stroke: $primary-color;
+        fill: $primary-color;
       }
     }
   }
 
+  .node-text {
+    cursor: pointer;
+    user-select: none;
+
+    &.selected {
+      fill: $light-color;
+      stroke: $light-color;
+    }
+  }
+
+  .edge-text {
```

Comment:
Why is `.edge-text` moved outside of `.edge`? Is the element now residing outside of the edge element? We might want to keep it where it is because it improves specificity.

Reply:
The purpose of this PR is to reduce specificity (see reasons laid out in  https://github.com/uber/react-digraph/issues/110) 

If this project requires this kind of selector specificity, then maybe a better direction would be to  allow consumers to ignore the default stylesheet completely and use their own, rather than have to override 6 levels of selector specificity for things like color customization. I can look more into this direction if you prefer.

Reply:
Let me discuss this with the team. We might go the route of BEM in this case, since we don't want to collide with client selectors and you want to reduce specificity. BEM can accomplish both by making the child classes more unique ("react-digraph__graph").